src/Calculations/Addition.cs

```diff
@@ -2,26 +2,11 @@ namespace Calculations;
 
 public class Addition : IAddition
 {
-    private readonly IUserConsoleInputValidation _validateInput;
-
-    public Addition(IUserConsoleInputValidation validateInput)
-    {
-        _validateInput = validateInput;
-    }
-
     public double FirstNumber { get; private set; }
```

Comment:
bitpick: seems, these properties are not required anymore

Reply:
Yes, I will get rid of them.